lib/libutee/sub.mk

```diff
@@ -12,6 +12,8 @@ srcs-y += tee_api_objects.c
 srcs-y += tee_api_operations.c
 srcs-y += tee_api_se.c
 srcs-y += tee_api_panic.c
+srcs-y += tee_tcpudp_socket.c
+srcs-y += tee_socket_pta.c
```

Comment:
OK, makes sense. I was a bit concerned with increasing the TA code size, but in principle unused code should be discarded by the linker since the API is fully contained in its own compilation unit.